src/PerformanceTests/Common/Statistics/SimpleStatisticsFeature.V6.cs

```diff
@@ -57,7 +57,7 @@ public class Collector : StatisticsBehavior.Implementation
             private static readonly long TimeUnit = Stopwatch.Frequency;
 
             private readonly string _titleFormat;
-            private readonly bool _outputTitle;
+            private readonly bool _outputTitle = Environment.OSVersion.Platform == PlatformID.Win32NT;
```

Comment:
I think this also needs `&& Environment.UserInteractive` as you did elsewhere. It probably is a bug that it previously was always set to true.

Reply:
@ramonsmits do we ever run those tests in non-interactive mode? I'm asking for me learning

Reply:
what would happen if you would set the title when not in user interactive mode?

Reply:
@danielmarbach I don't think so, I don't see the need to run this as a windows service. This code originally is from another project.

@timbussmann Trying to set the console results in an exception as there is no console when you run as a windows service.

Reply:
@ramonsmits given that explanation, I think the current code should be fine then?